backend/howtheyvote/scrapers/votes.py

```diff
@@ -912,3 +913,109 @@ def _extract_data(self, doc: BeautifulSoup) -> Fragment | None:
                 "geo_areas": geo_areas,
             },
         )
+
+
+class OEILSummaryIDScraper(BeautifulSoupScraper):
```

Comment:
It would be nice to also port the test cases from the old codebase.

Reply:
Agreed, I honestly forgot about them, that would have been doable. Imo, this would not be necessary for a first merge.